### PR TITLE
Add description in qos-pod-4.yaml

### PR DESCRIPTION
**Pull request by ydFu**
Add description that distinguish between container-1 and container-2 make it easy to read.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by ydFu** [comment]
@sftim 
Thank for review this PR.

**Comment by ydFu** [comment]
/assign @zparnold
Thanks.

File: content/en/examples/pods/qos/qos-pod-4.yaml
Comment (Aut0R3V) [inline content/en/examples/pods/qos/qos-pod-4.yaml:8]: I think the added comments make sense. Maybe an existing member can comment for cofirmation.

**Review by Aut0R3V — APPROVED** [review]

File: content/en/examples/pods/qos/qos-pod-4.yaml
Comment (sftim) [reply content/en/examples/pods/qos/qos-pod-4.yaml:8]: This PR already has an LGTM. With another LGTM I can approve it; I would like a second review from a Kubernetes member before this merges.

Comment (ydFu) [reply content/en/examples/pods/qos/qos-pod-4.yaml:8]: /assign @sftim 
l thinked this PR be reviewed from other Kubernetes member(@Aut0R3V),
Could you help to add "approved label" ?
Thanks for your help.

/cc @Aut0R3V @rajeshdeshpande02 @steveperry-53
Could you help to revirew this PR ?
Thanks for your help.

**Comment by ydFu** [comment]
/assign @tengqm 
Please help review this PR.
Let me know if there is anything need to be revised.

File: content/en/examples/pods/qos/qos-pod-4.yaml
Comment (irvifa) [reply content/en/examples/pods/qos/qos-pod-4.yaml:8]: I’m not sure if we need a comment about the limit provided to the memory since it’s already defined on the manifest though.

Comment (irvifa) [reply content/en/examples/pods/qos/qos-pod-4.yaml:8]: Otherwise it’s lgtm

**Comment by tengqm** [comment]
@ydFu TBH, I don't think we really need comments like these. The manifest is already straightforward.

**Comment by ydFu** [comment]
Thanks for your guidance.
At the time I thought the code in `qos-pod-4.yaml` of this article in `configure-pod-container/quality-service-pod.md` that is for 
introduction to getting started,so just added description to replace the special blank line.

**Comment by tengqm** [comment]
@ydFu All the YAML manifests under the `examples` directory are associated with some contexts. These examples are provided to save the users some time on manually creating every YAML file. In this case, I'd prefer leaving some room for users to digest the manifests. Comments are only necessary when something unusual or something worth noting.

**Comment by ydFu** [comment]
@tengqm 
Thank you for your instruction and advice.
It let me know the norms in the YAML manifests under the `examples` directory.
The next submission is similar the adjustment,I will pay special attention.

**Pull request closed without merging**
